options.mk

```diff
@@ -247,6 +247,9 @@ ifeq ($(WOLFTPM),1)
   endif
   WOLFCRYPT_OBJS+=./lib/wolfssl/wolfcrypt/src/aes.o
   WOLFCRYPT_OBJS+=./lib/wolfssl/wolfcrypt/src/hmac.o
+  ifeq ($(DEBUG),1)
+    CFLAGS+=-DTPM_DEBUG=1
```

Comment:
We use `DEBUG_TPM` in wolfTPM.

Reply:
we renamed it in `WOLFBOOT_DEBUG_TPM`

Reply:
we renamed it to `WOLFBOOT_DEBUG_TPM` as suggested